terraform/modules/pubsub.tf

```
@@ -0,0 +1,44 @@
+data "google_pubsub_topic" "workflow_topic" {
+  name    = "workflow-requests"
+  project = var.project_id
+}
+
+data "google_pubsub_topic" "workflow_topic_dlq" {
+  name    = "workflow-requests-dead-letter"
+  project = var.project_id
+}
+
+resource "google_pubsub_subscription" "run_from_pubsub" {
+  name    = "${var.application}-sub"
+  project = var.project_id
+  topic   = data.google_pubsub_topic.workflow_topic.name
+
+  filter = <<EOF
+    attributes.integration_type = "${replace(var.application, "workflow-", "")}"
```

The attribute should be caled `workflow_name`
```suggestion
    attributes.workflow_name = "${replace(var.application, "workflow-", "")}"
```